frontend/src/modules/EditZing/Components/EmailPreview.tsx

```diff
@@ -0,0 +1,57 @@
+import { getBody } from '../utils/emailTemplates'
+import { EmailPreviewProps } from 'EditZing/Types/ComponentProps'
+import { Box } from '@mui/material'
+import { Typography } from '@mui/material'
+import { SxProps } from '@mui/material'
+
+export const EmailPreview = ({
+  templateName,
+  courseNames,
+}: EmailPreviewProps) => {
+  const TitleSx: SxProps = {
+    color: 'essentials.6',
+    backgroundColor: 'essentials.75',
+    borderBottom: '0.5px solid #898992;',
+    border: '0.5px solid',
+    borderColor: 'essentials.50',
+    padding: '16px',
+    fontWeight: '900',
+    borderRadius: '5px 5px 0px 0px;',
+  }
+  const BodyAndSubjectSx: SxProps = {
+    color: 'essentials.75',
+    backgroundColor: 'essentials.6',
+    border: '0.5px solid',
+    borderBottom: '0.5px solid #898992;',
+    borderColor: 'essentials.50',
+    padding: '16px',
+  }
```

Comment:
Alternatively you could pass it a function `theme => ({ ... })` instead of just the object if you want to take advantage of things like `padding: 2` which multiplies the theme's default spacing by 2 (default spacing is 8). Don't need to do this though